lib/command.ts

```diff
@@ -184,7 +183,7 @@ export default class Command implements ICommand {
     this.replyEncoding = options.replyEncoding;
     this.errorStack = options.errorStack;
 
-    this.args = flatten(args);
+    this.args = [].concat(...args);
```

Comment:
This seems not to be a equivalent transformation.

Reply:
Should be probably according to

https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_flatten

this.args = args.reduce( (a, b) => a.concat(b), []);